catalogs/1.0.0.json

```diff
@@ -367,6 +367,14 @@
             "description": "Adds a brand new, nomadic-style Vasari faction.",
             "url": "https://github.com/martijnlv40/Vasari-purists",
             "versions": {
+				"1.3.0": {
+                    "download": "https://github.com/martijnlv40/Vasari-purists/archive/refs/tags/1.3.0.zip",
+                    "sinsVersion": "1.16.10.0",
+                    "greedVersion": "2.0.0",
+                    "dependencies": [],
+                    "conflicts": [],
+                    "date": "12/07/2023"

```

Comment:
are you from the future?

Reply:
Lol that stupid American date notation messed with me